src/views/Habits.vue

```diff
@@ -94,19 +94,20 @@ a {
 
 .center {
   text-align: center;
+  margin: 5% auto;
 }
 
 @media (min-width: 500px) {
   .habits {
     background-image: url("../assets/other-1.png");
     margin: auto;
-    background-size: 50% auto;
+    background-size: 25% auto;
+    height: -webkit-fill-available;
```

Comment:
Try to avoid using this value as it's specific to only Webkit-based browsers (Chrome, Safari), and might not work on other browsers (Firefox, Edge).

Reply:
Ah, ok got it! Thanks